hack/versions/pkg/version/version.go

```diff
@@ -32,9 +32,20 @@ import (
 func GetLatestVersion() (string, bool) {
 	current := strings.TrimPrefix(latest.Version, "skaffold/")
 	logrus.Infof("Current Skaffold version: %s", current)
-	logrus.Infof("checking for released status of %s...", current)
+
+	config, err := ioutil.ReadFile("pkg/skaffold/schema/latest/config.go")
+	if err != nil {
+		logrus.Fatalf("failed to read latest config: %s", err)
+	}
+
+	if strings.Contains(string(config), "This config version is already released") {
```

Comment:
Can you look for `hack/versions/pkg/schema/comment.go#releasesComment` instead of this string. 
`./hack/release.sh` actually add the comment with this constant.

Reply:
Done. Thanks for the feedback!